packages/create-cli/src/utils/directory.ts

```diff
@@ -14,8 +15,17 @@ export function hasGitDir () {
 export function copyTemporaryFiles (dirPath: string) {
   const FILE_TO_KEEP = ['__checks__', 'checkly.config.ts']
 
-  for (const file of FILE_TO_KEEP) {
-    fs.renameSync(`${dirPath}/${file}`, `./${file}`)
+  if (FILE_TO_KEEP.some(file =>
+    fs.existsSync(path.join(process.cwd(), file)))) {
+    // eslint-disable-next-line no-console
+    console.error('It looks like you already have "__checks__" folder or "checkly.config.ts". ' +
+      'Please, remove them and try again.')
+    fs.rmSync(dirPath, { recursive: true })
+    process.exit(1)
```

Comment:
Please throw an error here and do the deletion and process.exit where this function is called

Reply:
Never mind, this code has a lot of problems